src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -15,6 +15,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
-
+        
```

Comment:
Add a comment if why this doesn't need to do anything or a todo if it will be filled in later.
i.e //TODO(<owner ldap>): <msg>

Reply:
Actually don't need it for now so will delete